ci/ci_openshift.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+TEST_RESULT=0
+
+function download_oc_client {
+  echo "==== Installing OC Client ===="
+  if [ -f ./oc ]; then
+    echo "oc client installed"
+  else
+    wget -q -N https://github.com/openshift/origin/releases/download/v1.3.0-alpha.2/openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
+    tar -zxf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
+    cp openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit/oc .
+    rm -rf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit
+    rm -rf openshift-origin-client-tools-v1.3.0-alpha.2-983578e-linux-64bit.tar.gz
+  fi
+}
+
+function start_cluster {
+  echo "==== Starting up cluster ===="
+  ./oc cluster up
+}
+
+function create_application {
+  echo "==== Creating Infinispan application ===="
+  ./oc new-app jboss/infinispan-server
```

Comment:
Are we going to test the image that is pushed on dockerhub? If we want to test PR changes to the image on Openshift, we'd need to build the image and push it to Openshift internal registry